docs/diagrams/ModelClassDiagram.puml

```diff
@@ -42,13 +47,24 @@ Person *--> Phone
 Person *--> Email
 Person *--> Address
 Person *--> "*" Tag
+Person *--> Age
+Person *--> Sex
+Person *--> StarredStatus
+Person *--> "*" Appointment
+Person *--> Note
 
 Person -[hidden]up--> I
 UniquePersonList -[hidden]right-> I
 
 Name -[hidden]right-> Phone
 Phone -[hidden]right-> Address
 Address -[hidden]right-> Email
+Email -[hidden]right-> Tag
+Tag -[hidden]right-> Age
+Age -[hidden]right-> Sex
+Sex -[hidden]right-> StarredStatus
+StarredStatus -[hidden]right-> Appointment
+Appointment -[hidden]right-> Note
 
```

Comment:
Would it be better to adjust the size of the diagram such that it fits with the other diagrams in our developer guide?